Directory.Build.props

```diff
@@ -73,9 +70,4 @@
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
   </PropertyGroup>
 
-  <!-- Test config -->
-  <PropertyGroup>
-    <UseVSTestRunner>true</UseVSTestRunner>
```

Comment:
I recall I had some issue with xunit when this entry was not around but let's see.

Reply:
This was just moved into Versions.props where Arcade feature flags are expected to be located.